### PR TITLE
PriceList_Changes

### PR DESCRIPTION
**Pull request by kumarp-v**
Price List - Valid From date changes

### CODE REVIEW

File: VIS/Areas/VIS/Models/Callouts/MPriceListVersionModel.cs [hunk 1]
```diff
@@ -26,8 +57,134 @@ public int GetM_PriceList_Version_ID(Ctx ctx, string fields)
...
+                _columnName = "StartDate";
+            }
+            else
+            {
+                _tableName = "C_Order";
+                _keyColumnName = "C_Order_ID";
+                _columnName = "DateOrdered";
+            }
+        }
+
```
Comment (Manjot-Vienna) [inline VIS/Areas/VIS/Models/Callouts/MPriceListVersionModel.cs:110]: Please do commenting on functions and changes you did

**Review by Manjot-Vienna — CHANGES_REQUESTED** [review]
Please do commenting

**Review by Manjot-Vienna — DISMISSED** [review]
OK

File: VIS/Areas/VIS/Models/Callouts/MPriceListVersionModel.cs [hunk 2]
```diff
@@ -61,13 +65,24 @@ public int GetM_PriceList_Version_ID(Ctx ctx, string fields)
             //return Util.GetValueOfInt(DB.ExecuteScalar(sql));
         }
 
+        /// <summary>
+        /// Get Price List Version on ValidFrom Date
+        /// </summary>
+        /// <param name="ctx"></param>
+        /// <param name="fields"></param>
+        /// <param name="transactionType"></param>
```
Comment (Manjot-Vienna) [inline VIS/Areas/VIS/Models/Callouts/MPriceListVersionModel.cs:73]: add parameter names and what that method will return Return

File: VIS/Areas/VIS/Models/Callouts/MPriceListVersionModel.cs [hunk 3]
```diff
@@ -107,7 +122,17 @@ private void SetTableAndColumnName(TransactionType transactionType)
...
             }
         }
-        
+
+        /// <summary>
+        /// Get Price List Version on ValidFrom Date
+        /// </summary>
+        /// <param name="ctx"></param>
+        /// <param name="priceListId"></param>
+        /// <param name="transactionId"></param>
```
Comment (Manjot-Vienna) [inline VIS/Areas/VIS/Models/Callouts/MPriceListVersionModel.cs:131]: same in this function as well

**Review by Manjot-Vienna — APPROVED** [review]
OK

**Review by Amit-Vienna — APPROVED** [review]
ok

**Pull request merged**